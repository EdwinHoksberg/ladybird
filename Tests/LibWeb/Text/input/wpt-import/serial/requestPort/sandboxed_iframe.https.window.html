<!doctype html>
<meta charset=utf-8>

<script src="../../resources/testharness.js"></script>
<script src="../../resources/testharnessreport.js"></script>
<script src="../../resources/testdriver.js"></script>
<script src="../../resources/testdriver-vendor.js"></script>
<div id=log></div>
<script src="../../serial/requestPort/sandboxed_iframe.https.window.js"></script>
