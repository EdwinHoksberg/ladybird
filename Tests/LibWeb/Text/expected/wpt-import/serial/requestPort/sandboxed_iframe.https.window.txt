Harness status: OK

Found 1 tests

1 Fail
Fail	RequestPort from a sandboxed iframe is valid.